src/js/components/validation.js

```diff
@@ -29,8 +32,7 @@ class Validation {
       this.validate(e.target)
     }
 
-    this.$element.off(this.options.events, this.options.selector, this.handler)
-    this.$element.on(this.options.events, this.options.selector, this.handler)
+    registerEvents(this._fields, this._events, this.handler)
```

Comment:
Voce pode pegar o `options.events` e `options.selector` por aqui, dai voce nao precisa criar o `this` pra eles no init

Reply:
De qualquer forma eu precisaria passar um array de elementos aqui, então mesmo usando direto o `options.selectors` eu teria que fazer esse _setup_ dos seletores, o que combina mais com o método `init` do que com o `bindListeners`.